leaderboard/models.py

```diff
@@ -756,20 +988,18 @@ def _compute_score(self):
         ref_text_path = self.get_ref_text(path_only=True)
 
         try:
-            hyp_stream = (x for x in open(hyp_text_path, encoding='utf-8'))
-            ref_stream = (r for r in open(ref_text_path, encoding='utf-8'))
+            hyp_stream = [x for x in open(hyp_text_path, encoding='utf-8')]
```

Comment:
@snukky: just curious, why change to list comprehension?

Reply:
Hm. First, I've changed it to lists to be able to print their values when debugging. Then I decided to leave them for the sake of removing two duplicated lines below. But I haven't thought much if there are other potential issues from this change, e.g. optimization-wise.